ci/images/debian10/Dockerfile

```diff
@@ -24,25 +24,17 @@ RUN apt-get install -y gettext-base
 RUN apt-get install -y git rsync unzip jq
 
 # Installs shellcheck.
-RUN curl -fsSL https://github.com/koalaman/shellcheck/releases/download/v0.7.1/shellcheck-v0.7.1.linux.$(uname -m).tar.xz | \
-    tar -xJ && \
-    mv shellcheck*/shellcheck /usr/local/bin && \
-    rm -R shellcheck*
-
-# Install Go.
-RUN ARCH="$(uname -m | sed 's/x86_64/amd64/; s/aarch64/arm64/')" && \
-    curl -fsSL "https://dl.google.com/go/go1.14.3.linux-$ARCH.tar.gz" | tar -C /usr/local -xz
-ENV GOPATH=/gopath
-# Ensures running this image as another user works.
-RUN mkdir -p $GOPATH && chmod -R 777 $GOPATH
-ENV PATH=/usr/local/go/bin:$GOPATH/bin:$PATH
-
-# More stable than go get
-RUN curl -sfL https://install.goreleaser.com/github.com/goreleaser/nfpm.sh | sh
-
-RUN VERSION="$(curl -fsSL https://storage.googleapis.com/kubernetes-release/release/stable.txt)" && \
-    curl -fsSL "https://storage.googleapis.com/kubernetes-release/release/$VERSION/bin/linux/amd64/kubectl" > /usr/local/bin/kubectl \
-    && chmod +x /usr/local/bin/kubectl
+RUN curl -fsSL https://github.com/koalaman/shellcheck/releases/download/v0.7.1/shellcheck-v0.7.1.linux.$(uname -m).tar.xz \
```

Comment:
I think there's a Debian package for shellcheck, so you can apt-get install shellcheck here? The current stable release is a little old though: https://packages.debian.org/buster/shellcheck unless you also enable backports: https://packages.debian.org/buster-backports/shellcheck

No worries though, this is fine as it is :)

Reply:
We now actually grab shellcheck from npm now, which uses a script that directly downloads/installs it to `node_modules` - see https://github.com/cdr/code-server/blob/main/package.json#L71 - I landed this in the CI rewrite to make building easier

Reply:
nice! so we can remove the curl line?

I was confused that shellcheck in package.json is v1 while the latest release in GitHub is 0.7.1, and it seems that the npm package just downloads the latest release? This would make the build less reproducible, since shellcheck can can change (and break) without any changes on our part

Reply:
Oh huh yeah good point. I think we wanted to use the `npm` version to reduce the number of dependencies users have to manually install so maybe we can fork it, make the version static, and publish `@coder/shellcheck` or something.

Reply:
Could we convince the shellcheck people to release versioned releases that correspond with the upstream versions? e.g. shellcheck@0.71-1 or something (upstream version 0.71, first version of the npm package) - that would be useful for other people too and means we don't have to maintain our own thing

Reply:
Maybe we can do the same for `nfpm`.

Reply:
That would definitely be better if we could convince upstream! Seems the maintainer is open to proposals: https://github.com/gunar/shellcheck/issues/24#issuecomment-684924783

Reply:
My main concern was that since 1.0.0 was already published it seems a bit sketchy.